modules/rntuple.mjs

```diff
@@ -609,15 +665,18 @@ class RNTupleDescriptorBuilder {
 
     // Example Of Deserializing Page Content
     deserializePage(blob, columnDescriptor) {
-        const reader = new RBufferReader(blob),
-            values = [],
-            coltype = columnDescriptor.coltype,
+        const {
+            blob: processedBlob,
+            coltype
+        } = recontructUnsplitBuffer(blob, columnDescriptor),
             byteSize = getTypeByteSize(coltype),
-            numValues = byteSize ? blob.byteLength / byteSize : undefined;
+            reader = new RBufferReader(processedBlob),
+            values = [],
+            numValues = byteSize ? processedBlob.byteLength / byteSize : undefined;
```

Comment:
I don't think it makes sense to continue running if `byteSize` is not defined - you should throw rather than running the loop on `processedBlob.byteLength`